lexicons/fyi/frontpage/richtext/block.json

```diff
@@ -0,0 +1,35 @@
+{
+  "lexicon": 1,
+  "id": "fyi.unravel.frontpage.richtext.block",
+  "defs": {
+    "main": {
+      "type": "union",
+      "refs": ["#paragraph"]
+    },
+    "paragraph": {
+      "type": "object",
+      "properties": {
+        "content": { "type": "ref", "ref": "#richtext" }
+      }
+    },
+    "richtext": {
```

Comment:
This type needs some work, we need to solve for overlapping spans.

https://github.com/bluesky-social/atproto/discussions/2830#discussioncomment-10759975

https://github.com/likeandscribe/frontpage/pull/108#discussion_r1694220823

Reply:
I don't really want to couple specifying rich text to this PR, so I may end up adding a `plaintextParagraph` type that's equivalent to our current post text field. Then we can add rich text later.